perfkitbenchmarker/benchmark_spec.py

```diff
@@ -157,7 +157,7 @@ def UnPickleLock(locked, *args):
     'os_type is "rhel". In general if two OS\'s use the same package manager, '
     'and are otherwise very similar, the same os_type should work on both of '
     'them.')
-flags.DEFINE_string('scratch_dir', '/scratch',
+flags.DEFINE_string('scratch_dir', None,
```

Comment:
No - the default is to use the value in default_config_constants.yaml, which is still /scratch.